common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -108,6 +108,7 @@ def _student_or_public_view(self, context, view):
             'xblock_context': context,
             'unit_title': self.display_name_with_default if not is_child_of_vertical else None,
             'due': self.due,
+            'due_num_weeks': self.due_num_weeks,
```

Comment:
What's this for?

Reply:
I initially thought this was where the vertical block's information was written but I was wrong! Removed